docs/components/aws/elasticache.md

```diff
@@ -0,0 +1,792 @@
+---
+description: AWS ElastiCache, in-memory data storage and cache services for your serverless functions, with SLAppForge Sigma cloud IDE
+---
+
+# ![](images/elasticache/ecache_icon.svg) AWS ElastiCache
+
+ElastiCache is a fully managed, in-memory data store for enterprise-level, data-intensive applications. ElastiCache 
+can improve the performance of consumer applications, by retrieving data from its in-memory data stores with high throughput and low latency. 
+ElastiCache supports two open-source cache engines, Amazon ElastiCache for Redis and Amazon ElastiCache for Memcached.
+
+The Sigma IDE currently supports only for Amazon ElastiCache for Redis.
+
+## Amazon ElastiCache for Redis
+
+**Re**mote **Di**ctionary **S**erver is an open-source, in-memory key-value data storage that can offer multi-purpose 
+functionalities such as a database, cache, message broker, and queue. Sigma IDE provides the ability to create a new Redis cluster as well as 
+choose an existing cluster to execute operations.
+
+### New Redis Cluster
+
+To create a new cluster, first, go to the **New Cluster** tab of the configuration panel. Inorder to create new Redis cluster following configurations 
+required.
+
+#### Cluster Engine Configuration
+
+<p align="center">
+  <img style="Padding:20px" width="400" src="./images/elasticache/ecache_cluster_conf.png">
+</p>
+
+Field                       | Required           | Description
+---                         | :---:              | ---
+Engine                      | :white_check_mark: | The cache engine type to be used for the cluster   
+Engine Version              | :white_check_mark: | The version number of the cache engine to be used for the cluster 
+Node Type                   | :white_check_mark: | The compute and memory capacity of the node
+Cluster mode                | :white_check_mark: | Standalone or Clustered 
+Multi-AZ with Auto-failover | :white_check_mark: | Auto failover with multi availability zones
+Number of Shards            | :white_check_mark: | Number of shards in the cluster 
+Number of Replicas          | :white_check_mark: | Number of replicas within a single shard
+&nbsp;
+
+Field                       | Constraints
+---                         | ---
+Multi-AZ with Auto-failover | Only available in standalone mode(cluster mode disabled). In cluster mode enabled by default
+Number of Shards            | Only available in cluster mode, minimum:2 maximum:15. Otherwise, default to be 1  
```

Comment:
mode_; minimum: 2, maximum: 15_.

Drop "Otherwise", "be" (just "Defaults to 1.")